tutorials/peft-curation-with-sdg/config/sem_dedup_config.yaml

```diff
@@ -0,0 +1,32 @@
+# Configuration file for semdantic dedup
+cache_dir: "_temp/semdedup_cache"
+num_files: 16
+id_col_name: "id"
+id_col_type: "str"
+input_column: "text"
+
+# Embeddings configuration
+embeddings_save_loc: "embeddings"
+embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
+embedding_batch_size: 128
+embedding_max_mem_gb: 20
+
+# Clustering configuration
+clustering_save_loc: "clustering_results"
+n_clusters: 1000
```

Comment:
You should lower this to like `1` in this case since the dataset is super small in size. This is for 2 reasons.

1. `n_clusters` being larger reduces the computational burden, but it also reduces the accuracy of catching duplicates.
2. I seem to be running out of file descriptors when `n_clusters` is this large. I'll check with @VibhuJawa on this but for now setting it lower solves this issue.

Reply:
I didn't run into the file descriptors issue you mentioned. @VibhuJawa let me know what you think is best to set this to, considering the other settings I have in the config for SemDedupe.

Reply:
I think we can bring it down to `n_clusters` = `10/100` . 

@avem-nv  do you have any guidance when we are running a workflow on just `12,224` documents. I think reducing the number should help catch duplicates better.

Reply:
Merging this in now due to time sensitivity. We can adjust later.